src/components/ModalFlows/ChallengeProposalModal/ChallengeProposalModal.js

```diff
@@ -0,0 +1,44 @@
+import React, { useMemo, useState, useCallback } from 'react'
+import MultiModalScreens from '../../MultiModal/MultiModalScreens'
+import ChallengeBalances from './ChallengeBalances'
+import TransactionStepper, { modalWidthFromCount } from '../TransactionStepper'
+import { useMounted } from '../../../hooks/useMounted'
+
+function ChallengeProposalModal() {
+  const mounted = useMounted()
+  const [transactions, setTransactions] = useState([])
+
+  const challengeProposal = useCallback(
+    async (onComplete) => {
+      try {
+        if (mounted()) {
+          setTransactions([])
+        }
```

Comment:
We can set the transactions here after receiving them via the connector, they then get passed to the Stepper in the next screen which will prompt the signing/s